mne/bem.py

```diff
@@ -1359,3 +1361,274 @@ def write_bem_solution(fname, bem):
                                bem['solution'])
         end_block(fid, FIFF.FIFFB_BEM)
         end_file(fid)
+
+
+# #############################################################################
+# Create 3-Layers BEM model from Flash MRI images
+
+def make_flash_bem(subject, subjects_dir, no_flash30=False, no_convert=False,
+                   unwarp=False, overwrite=False, show=False):
+    """Create 3-Layers BEM model from Flash MRI images
+
+    Parameters
+    -----------
+    subject : str
+        Subject name.
+    subjects_dir : str
+        Directory containing subjects data (Freesurfer SUBJECTS_DIR).
+    no_flash30 : bool
+        Skip the 30-degree flip angle data.
+    no_convert : bool
+        Assume that the Flash MRI images have already been converted
+        to mgz files.
```

Comment:
In fact, in Python land I'd say we should split the unwarping step into a separate function entirely...